src/client/delphi_epidata.py

```diff
@@ -15,19 +15,12 @@
 from tenacity import retry, stop_after_attempt
 
 from aiohttp import ClientSession, TCPConnector, BasicAuth
-from importlib.metadata import version, PackageNotFoundError
 
 from delphi.epidata.common.logger import get_structured_logger
 
-# Obtain package version for the user-agent. Uses the installed version by
-# preference, even if you've installed it and then use this script independently
-# by accident.
-try:
-    _version = version("delphi-epidata")
-except PackageNotFoundError:
-    _version = "0.script"
+_version_ = "4.1.20"
```

Comment:
still doesnt match whats in `__init__.py`!
```suggestion
__version__ = "4.1.20"
```
the convention is for this to be a "dunder", as seen in the good ol:
```py
if __name__ == '__main__':
```